JavascriptFormatter.js

```diff
@@ -475,13 +472,10 @@ CallSelenium.prototype.invert = function () {
 };
 CallSelenium.prototype.toString = function () {
     log.info('Processing ' + this.message);
-    if (this.message == 'waitForPageToLoad') {
-        return '';
-    }
     var result = '';
     var adaptor = new SeleniumWebDriverAdaptor(this.rawArgs);
-    if (this.message == 'getEval')
-        adaptor.rawArgs = this.args; // getEval only args available
+    if (this.message.match(/^(getEval|runScript)/))
+        adaptor.rawArgs = this.args; // getEval only args available (Daniel: I assume this means we always want escaped stringified args here; that is, the code as converted to a string, for use with browser.safeEval(<code string>), and getEval may be used elsewhere, so we still want to have that pass forth unescaped rawArgs by default...?)
```

Comment:
This is a bug. getEval rawArgs is undefined and args has its value. Here just assign it back